package.json

```diff
@@ -39,7 +39,7 @@
     "@babel/core": "^7.22.11",
     "@khanacademy/flow-to-ts": "^0.5.2",
     "@napi-rs/cli": "^2.15.2",
-    "@parcel/babel-register": "2.10.1",
+    "@parcel/babel-register": "*",
```

Comment:
@mischnic @devongovett not sure if we wanted to include this change in the patch, and if this needs to be included in the CHANGELOG since its only in the root package.json?